homeassistant/components/mealie/quality_scale.yaml

```diff
@@ -0,0 +1,81 @@
+rules:
+  # Bronze
+  action-setup: done
+  appropriate-polling: done
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: todo
```

Comment:
We have a config flow

Reply:
Yes, but we are missing data_descriptions

Reply:
Ah, let me raise a PR for that.

Reply:
PR raised https://github.com/home-assistant/core/pull/133590